app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -444,7 +451,7 @@ export default class SampleDetails extends React.Component {
 
     const saveAndCloseBtn = belongToReaction && !sample.isNew ? this.saveBtn(sample, true) : null;
     return (
-      <div className='d-flex gap-1'>
+      <div className="d-flex gap-1 mt-3">
```

Comment:
This margin should be handled by tabs-container--with-borders. Let's checkout why it's missing together.